ingestors/pypi_rss.go

```diff
@@ -0,0 +1,63 @@
+package ingestors
+
+import (
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+)
+
+//const pyPiPackagesFeedUrl = "https://pypi.org/rss/packages.xml"
+const pyPiUpdatesFeedUrl = "https://pypi.org/rss/updates.xml"
+
+type PyPiRss struct {
```

Comment:
Any reason not to call it `PyPi`?

Reply:
Because we're going to have two

Reply:
https://app.clubhouse.io/tidelift/story/15803/build-a-xml-rpc-pypi-ingestor-for-depper